randovania/data/json_data/prime3.txt

```diff
@@ -1715,7 +2034,7 @@ Asset id: 9934831212540578712
 > Dock to Warp Site Alpha; Heals? False
```

Comment:
Not for Main Lift, but for Fiery Airdock: add node called "Samus Ship", with "Heals?" checked, and have it connect to the door as "Trivial"